src/BestBooks.js

```diff
@@ -76,15 +104,19 @@ class BestBooks extends React.Component {
                   status={book.status}
                   email={book.email}
                   onDelete={this.handleDelete}
+                  onUpdateModal={this.handleUpdateModal}
                   book={book}>
                 </Book>
               </Carousel.Item>
             );
           })}
           </Carousel>) : (<h3>No Books Found </h3>)}
 
+        
         {this.state.showBookForm ? <BookFormModal onCreate={this.handleCreate} /> : <AddBookButton onButtonClick={this.showBookFormHandler} />}
 
+        <UpdateBook book={this.state.selectedBook} onUpdate={this.handleUpdate} onClose={() => this.setState({ selectedBook: null })} />
+
```

Comment:
Great work here!

Just a tid bit of feedback: 

For setState calls, decide which way you want to do these, the shorthand version as seen here, or the other way which you've done on lines 57/51 for `handleUpdate()` and `handleUpdateModal()`(by creating a method and then passing method/handler as prop). Try and stick with one way for consistency if ya can 👍